lib/cryptoexchange/exchanges/bitshares_assets/services/market.rb

```diff
@@ -0,0 +1,35 @@
+module Cryptoexchange::Exchanges
+  module BitsharesAssets
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          output = super(ticker_url)
+          adapt(output, market_pair)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::BitsharesAssets::Market::API_URL}/asset/markets?asset=BTS"
+        end
+
+        def adapt(output, market_pair)
+          ticker           = Cryptoexchange::Models::Ticker.new
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = BitsharesAssets::Market::NAME
+
+          ticker.last      = NumericHelper.to_d(output['OPEN.BTC']['price'])
```

Comment:
Doing this would mean that it is only taking a value for `OPEN.BTC`, this shouldnt be hardcoded since there are many other currencies in the response